tools/autograd/gen_variable_type.py

```diff
@@ -248,7 +249,7 @@
 
 SETUP_ANY_REQUIRES_GRAD = CodeTemplate("""\
 auto _any_requires_grad = compute_requires_grad( ${args_with_derivatives} );
-(void)_any_requires_grad;
+${extra_differentiability_conditions}(void)_any_requires_grad;
```

Comment:
missing new line?

Reply:
The problem with adding a new line is that if `${extra_differentiability_conditions}` is '' (which it is most of the time), then there is an extra new line in all the generated code. Maybe that's not a bad thing, what are your thoughts?

Reply:
Ho right. I would say it is not the end of the world to have the extra empty line in the generated code. But it will look weird indeed.